specification/security/delete_role/SecurityDeleteRoleRequest.ts

```diff
@@ -21,6 +21,8 @@ import { RequestBase } from '@_types/Base'
 import { Name, Refresh } from '@_types/common'
 
 /**
+ * Delete roles API
```

Comment:
There should be an empty line between the header and the description.

The JSDoc is interpreted as markdown which means that no new paragraph is created without the empty line.

Same for all other places 🙂

Reply:
@flobernd thanks! 

Is this what you had in mind: https://github.com/elastic/elasticsearch-specification/pull/3033/commits/4ca237aae7af48fe35926c4614be4437d3ef3d81
?

Reply:
Hmm the `description` field is still borked: https://github.com/elastic/elasticsearch-specification/pull/3033/files#diff-4ec6052a151433c851f9e7b60e2ac3a6814b1ac6aa977c6c43bf61cebc976038L8382

Reply:
@n1v0lg Yes, the previous commit looks good! It should affect the ouput as well 😅 (maybe we have a bug here in the spec compiler, if it really does not..).